### ISSUE TITLE
​Self-hosted shouldn't say "Affine Cloud"

### ISSUE THREAD

**Issue by soudaburger** [toeverything/AFFiNE#7097]
### Description

When connecting a self-hosted instance, it should really not say "Affine Cloud".  It took me googling and finding some other post on Reddit, I think, to realize that it wasn't really "Cloud" connected, but going to connect the Postgres instance I configured already.

### Use case

Being able to specify an environment variable like "SELF_HOSTED=true" would allow the UI to inform the user that they aren't actually going to connect it to Affine's cloud, but rather the self-hosted infrastructure behind this instance.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by airdogvan** [comment]
I strongly agree with [soudaburger](https://github.com/soudaburger), this is extremely confusing, to the point that after verification, I'm still not clear whether my data is being kept locally or on your cloud!!

For self-hosters I think this is an urgent issue.  The only reason I'm trying Affine, apart from the very promising functionality,  is because of the possibility to keep my data to myself.

But let me be very clear, if I have no clear guarantee that my data is kept on my server, I'm out!

**Comment by dasrocky** [comment]
> I strongly agree with [soudaburger](https://github.com/soudaburger), this is extremely confusing, to the point that after verification, I'm still not clear whether my data is being kept locally or on your cloud!!
> 
> For self-hosters I think this is an urgent issue. The only reason I'm trying Affine, apart from the very promising functionality, is because of the possibility to keep my data to myself.
> 
> But let me be very clear, if I have no clear guarantee that my data is kept on my server, I'm out!

I have been using Trilium Notes user for a while and notion user for work. I have been looking for an alternative to migrate trillium notes to some other shelf-hosting note-taking solution. My strict requirement is a simple self-hosting note-taking system with no connection to external sources unless I want it.

I stumbled on Affine and Appflowy, I was amazed that they provide notion and drawing tools in the same software, maybe a possible alternative I think. A few weeks ago, When I tried their shelf-hosted option, I was so confused am connected to online affine cloud or my deployed cloud. I looked into other discussions, and I see similar problem or limited by 10 GB etc. Immediately removed the docker deployment. I think I wont look back into it again till they fix this and confirm that none of the data or telemetry or connection goes back. Switched to appflowy

**Comment by gymnae** [comment]
Same here. I was enticed with this product and the promise of self-hosting, but as soon as you looked closer red flags appeared: Unclear licensing, contradictory statements regarding the limits/freedom of self-hosting, hard coded third party dependencies etc etc. And the address in Singapore appears to be a go-to business center for when one needs to be registered in Singapore...

**Comment by Aliang-code** [comment]
> > I strongly agree with [soudaburger](https://github.com/soudaburger), this is extremely confusing, to the point that after verification, I'm still not clear whether my data is being kept locally or on your cloud!!
> > For self-hosters I think this is an urgent issue. The only reason I'm trying Affine, apart from the very promising functionality, is because of the possibility to keep my data to myself.
> > But let me be very clear, if I have no clear guarantee that my data is kept on my server, I'm out!
> 
> I have been using Trilium Notes user for a while and notion user for work. I have been looking for an alternative to migrate trillium notes to some other shelf-hosting note-taking solution. My strict requirement is a simple self-hosting note-taking system with no connection to external sources unless I want it.
> 
> I stumbled on Affine and Appflowy, I was amazed that they provide notion and drawing tools in the same software, maybe a possible alternative I think. A few weeks ago, When I tried their shelf-hosted option, I was so confused am connected to online affine cloud or my deployed cloud. I looked into other discussions, and I see similar problem or limited by 10 GB etc. Immediately removed the docker deployment. I think I wont look back into it again till they fix this and confirm that none of the data or telemetry or connection goes back. Switched to appflowy

yep, i think they are forgot why we come from here.
btw, the appflow also has too much bugs, even like could not edit kanban item unless enter the kanban viewpage😥.

**Comment by gymnae** [comment]
I switched to self-hosted outline and it does exactly what I want. It’s not overloaded with half-baked features, it just delivers a good UX.

Also, outline isn’t seo blog spamming like affine

**Comment by Aliang-code** [comment]
> I switched to self-hosted outline and it does exactly what I want. It’s not overloaded with half-baked features, it just delivers a good UX.
> 
> Also, outline isn’t seo blog spamming like affine

I haven't paid attention to outline before, so I will give a try

**Comment by Drun555** [comment]
It's just a beta right now. As long as Outline is a good thing too, AFFiNE is just has so much more to offer. I totally understand why selfhosted isn't a priority in UX, but I hope they will add features to disable bloated stuff like AI ads / this thing below / rename Affine Cloud

![image](https://github.com/user-attachments/assets/ef8e7b86-4c9e-43f8-ac3f-a5715ae6bdb5)

**Comment by cloudmaker97** [comment]
Oh my god, I've had this same trouble because I don't want to put my sensitive data in some cloud without knowing where, what encryption is used etc. But when "Cloud" means my self-hosted instance, I don't care. A rename of this product in self-hosted instances is appreciated!

**Comment by Wandering-Consciousness** [comment]
I totally agree and give 10+ votes!

**Comment by cielhaidir** [comment]
> It's just a beta right now. As long as Outline is a good thing too, AFFiNE is just has so much more to offer. I totally understand why selfhosted isn't a priority in UX, but I hope they will add features to disable bloated stuff like AI ads / this thing below / rename Affine Cloud
> 
> ![image](https://github.com/user-attachments/assets/ef8e7b86-4c9e-43f8-ac3f-a5715ae6bdb5)

This is really the cons of the self host, i hope this fix in the future

**Comment by cloudmaker97** [comment]
For everyone wondering @forehalo closing this without any referenced commit or pr; i've checked the [last commits from him](https://github.com/toeverything/AFFiNE/commits?author=forehalo&since=2025-03-31&until=2025-04-16) and neither found a hint for solving this issue. Does anyone have more information about it?

**Comment by gymnae** [comment]
It might be related to this commit: https://github.com/toeverything/AFFiNE/commit/f0fb1447a47d458d35c8fc7d7a92079a148c4c43
They are preparing pricing for self-hosting. 

Their picing page now states for the "free" self-host version to include cloud storage. It reads a bit like mandatory cloud connection even for self-hosted instances

> Included in Basic:
> 100 GB of Cloud Storage

**Comment by guyueshui** [comment]
> Their picing page now states for the "free" self-host version to include cloud storage. It reads a bit like mandatory cloud connection even for self-hosted instances

If that's the truth, I am not willing to use it anymore.

**Comment by gymnae** [comment]
> > Their picing page now states for the "free" self-host version to include cloud storage. It reads a bit like mandatory cloud connection even for self-hosted instances
> 
> If that's the truth, I am not willing to use it anymore.

I would not conclude that. Best read their current pricing, which offers clarity for self-hosting:
Hosted by yourself:
Unlimited local workspace
Unlimited use and Customization
Unlimited Doc and Edgeless editing
Included in Basic:
100 GB of Cloud Storage
100 MB of Maximum file size
Up to 10 members per Workspace
30-days Cloud Time Machine file version history
Community Support
Real-time Syncing & Collaboration for more people

**Comment by zakrok** [comment]
So enabling cloud does not allow us to use our private space but still uses the cloud? 
And at the same time if we don't enable it, the changes we make are only local in the browser and we cannot save it on our hosts?

**Comment by rennat** [comment]
So do I understand correctly: If I want real-time collaboration without my data going to or relying on a third party service to be functioning then this isn't the right tool.
